README.md

```diff
@@ -452,7 +448,7 @@ Then they are wrong, and we've not noticed yet. You can help by submitting a pul
 
 #### Can I change/enforce formatting settings for modules?
 
-No. We've explored trying to make DT's code-formatting consistent before but reached an impasse due to the high activity on the repo. We include formatting settings via a [`.editorconfig`](.editorconfig) and [`.prettierrc.json`](.prettierrc.json). These are exclusively for tooling in your editor, their settings don't conflict and we don't plan on changing them. Nor do we plan on enforcing a specific style in the repo. We want to keep the barriers to contributions low.
+No. We've explored trying to make DT's code-formatting consistent before but reached an impasse due to the high activity on the repo. We include formatting settings via a [`.editorconfig`](.editorconfig) and [`.dprint.jsonc`](.dprint.jsonc). These are exclusively for tooling in your editor, their settings don't conflict and we don't plan on changing them. Nor do we plan on enforcing a specific style in the repo. We want to keep the barriers to contributions low.
```

Comment:
this eventually needs more than a change of filenames.

Reply:
Oops, I forgot to actually read the sentence; will fix.

Reply:
Actually, is this not true? We're not allowing people to change formatting. Are you meaning that this should say how to opt-out for generated code?

Reply:
Yes, you can't change the settings. But it's more accurate to say that the whole repo is formatted with a tool, which is what the new paragraph says. Most of the rest either isn't relevant or isn't true anymore.

Reply:
Yeah, I just didn't read this properly. The ending is the bad part. We have a completely different entry for formatting that says "we use dprint", so I'm just going to replace it with the new one so it gets more reading.